Test/server/counterexample_none.transcript.expect

```diff
@@ -2,6 +2,6 @@
 
 Verifying Impl$$_module.__default.AbsCorrect ...
   [1 proof obligation]  verified
-COUNTEREXAMPLE_START {"States":[]} COUNTEREXAMPLE_END
+COUNTEREXAMPLE_START {"States":[{"Column":0,"Line":0,"Name":"<initial>","Variables":[]}]} COUNTEREXAMPLE_END
```

Comment:
I think we had to revert this change in the past.

Reply:
Done